websites/P/Pass Culture/metadata.json

```diff
@@ -0,0 +1,29 @@
+{
+	"$schema": "https://schemas.premid.app/metadata/1.9",
+	"author": {
+		"id": "807326854314590228",
+		"name": "Yuuto"
+	},
+	"service": "Pass Culture",
```

Comment:
Shouldn't it be a lowercase p here too? and same for the folder name, as its the official capitalization?

Reply:
oops you're right, let me fix that